.github/workflows/release-beta.yml

```diff
@@ -82,3 +82,41 @@ jobs:
           tags: ${{ env.DOCKER_HUB_TAG }}
           cache-from: type=registry,ref=${{ env.DOCKER_HUB_TAG }}
           cache-to: type=inline
+
+  jmeter:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_HUB_REPOSITORY: maestro-jmeter
+      DOCKER_HUB_TAG_PREFIX: beta
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+
+      - uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Get the version
+        id: get_version
+        run: echo "::set-output name=VERSION::$(git rev-parse --short HEAD)"
+
+      - name: Set Env Variables
+        run: |
+          echo "DOCKER_HUB_VERSION=${{ env.DOCKER_HUB_TAG_PREFIX }}-${{ steps.get_version.outputs.VERSION }}" >> $GITHUB_ENV
+          echo "DOCKER_HUB_TAG=${{ secrets.DOCKER_HUB_USERNAME }}/${{ env.DOCKER_HUB_REPOSITORY }}:${{ env.DOCKER_HUB_TAG_PREFIX }}-${{ steps.get_version.outputs.VERSION }}" >> $GITHUB_ENV
+
+      - name: Build and push
```

Comment:
We can test is this build passes by having branch started with `release-` so this workflow would be triggered

Reply:
The workflow has been tested in a released-* branch. We can now see the beta images at: https://hub.docker.com/r/farfetchoss/maestro-jmeter/tags

Thank you!